static/js/containers/ProfilePage.js

```diff
@@ -30,7 +31,10 @@ class ProfilePage extends ProfileFormContainer {
   stepTransitions: Function = (): [void|() => void, () => void] => {
     const { dispatch } = this.props;
     let setStep = createActionHelper(dispatch, setProfileStep);
-    let createStepFunc = step => () => setStep(step);
+    let createStepFunc = step => () => {
+      setStep(step);
+      ReactDOM.findDOMNode(this).querySelector(".profile-pagination").scrollIntoView();
```

Comment:
any particular reason you're using `ReactDOM` here instead of just `document.querySelector`?